common/src/main/res/values/strings.xml

```diff
@@ -1098,4 +1109,5 @@
     <string name="ha_assist">HA: Assist</string>
     <string name="only_favorites">Only Show Favorites</string>
     <string name="beacon_scanning">Beacon Monitor Scanning</string>
+    <string name="car_data_unavailable">Open Home Assistant app to enable CarInfo sensors</string>
```

Comment:
This shouldn't be used as a state, use `unavailable` like in other places (or maybe even don't change the state, just keep them as they were?)

Reply:
This was added to explicitly mention that the app needs to be started (https://github.com/home-assistant/android/pull/3399#issuecomment-1501993900), maybe we should keep it only for text sensors ?

Reply:
Missed that comment, sorry. The note could be more general/less technical, something like: Open Home Assistant app to activate this sensor.